mmpose/datasets/pipelines/shared_transform.py

```diff
@@ -27,11 +28,16 @@ class ToTensor:
         results (dict): contain all information about training.
     """
 
+    def __init__(self, device='cpu'):
+        self.device = device
+
     def __call__(self, results):
+        to_tensor = (lambda x: torch.from_numpy(x.astype('float32')).permute(
+            2, 0, 1).to(self.device) / 255.0)
```

Comment:
The `255.0` is a little bit hard-coded here. Is this always necessary? Because in the source code of `F.to_tensor`, 

https://github.com/pytorch/vision/blob/e9fb2a356384180d92dc60c49b4dd4b5b3c6ee8f/torchvision/transforms/functional.py#L152-L155

it seems that if the img is already located in [0.0, 1.0], it just returns the img tensor itself. 

BTW, `F.to_tensor` uses `div(255)`, I am not sure if this is faster. You may have a try!

Reply:
I've tested that in GPU and found the `division of 255.0` makes no difference in speed. The `division` is just similar to Normalization, which can also be speed up in GPU.

Reply:
I suggest using the in-place operation `dev_(255)` to save memory.